### PR TITLE
Fix users spinning box doc

### PR DESCRIPTION
**Pull request by melvinsoft**

### CODE REVIEW

File: docs/fix_tahoe_user.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+If after fix the user tokens, the problem persists, it's probably because the user isn't assigned to any site in edX.
+
+### Check the user organisation in AMC
+1. Go to `https://amc-app.appsembler.com/admin/organizations/organization/`
+2. Search for the Organisation by name, for example: *Redis* or *Cybereason*.
+3. Open the organisation and check the user is inside org, so it's basically inside the *Chosen Users* box. If is not, check for the user and add it.
+
+### Check for the user organisation in edX
+1. Go to `https://tahoe.appsembler.com/admin/sites/site/`
+2. Search for the Organisation by name, for example: *Redis* or *Cybereason*. This Django's admin page doesn't provide search box, so you can use the browser text search, and move yourself over the pages. There are not so many now, but we need to fix this ASAP.
```
Comment (thraxil) [inline docs/fix_tahoe_user.md]: I do see a search box on this page: 
sitesadmin

**Review by thraxil — APPROVED** [review]
This is great as a stopgap while we figure out how to fix the underlying problem.

File: docs/fix_tahoe_user.md (hunk 1 above)
Comment (melvinsoft) [reply docs/fix_tahoe_user.md]: @thraxil you're absolutely right, it's fixed.

**Comment by melvinsoft** [comment]
@thraxil Thanks! Squashed and merged!

**Pull request merged**